src/parsers/poscar.js

```diff
@@ -58,13 +58,13 @@ function toPoscar(materialOrConfig, omitConstraints = false) {
  * @param poscarFileContent
  * @returns {Number}
  */
-export function atomsCount(poscarFileContent) {
+export function poscarFileAtomsCount(poscarFileContent) {
```

Comment:
1. Maybe `getAtomsCount`
2. No need to add "poscar" or "xyz" again to the `getAtomsCount` functions (below) - we already isolated them to a file